.github/CONTRIBUTING.md

```diff
@@ -90,4 +92,4 @@ When updating or adding a new CF the test case url needs to be added to the json
 
 ## Recommendations
```

Comment:
Lost these completely?

also should add https://code.visualstudio.com/docs/editor/extension-marketplace#_workspace-recommended-extensions